src/components/editProfile/Form.css

```
@@ -0,0 +1,57 @@
+.panel-body {
+  position: absolute;
+  top: 0.6em;
+}
+
+.panel-header {
+  flex-direction: row;
+  justify-content: center;
+  display: flex;
+}
+
+.panel-header--body {
+  text-align: center;
+  border-bottom: 1px solid black;
+  padding-bottom: 2.5em;
+  width: 100%
+}
+
+.panel-header--title {
+  text-transform: uppercase;
+  letter-spacing: 0.1em; 
+}
+
+.panel-header--user-img {
+  height: 7em;
+  width: 8em;
+}
+
+.panel-header--user-subtitle {
+  letter-spacing: 0.1em;
+  margin-top: 0.3em;
+  margin-bottom: 0.5em;
+}
+
+.panel-header--user-links {
+  margin-top: 0px;
```

when using 0, there is no need to provide units. 
```suggestion
  margin-top: 0;
```